packages/types/src/utils.ts

```diff
@@ -78,7 +78,7 @@ export type SynchronousStreamablePayload<Produces> =
 
 export type SynchronousStreamable<Produces> = SynchronousStreamablePayload<Produces> & {
     contentType?: string,
-    topic?: string | string[]
+    topic?: string
```

Comment:
Why? If anything, `string[]` should be used.

Reply:
I saw that the way it was implemented didn't allow for more than one value

Reply:
I suggest removing it for now and bringing it back together with a fix in another PR.

Reply:
OK